consultation_analyser/consultations/jinja2/consultations/questions/show.html

```diff
@@ -21,12 +21,15 @@ <h1 class="govuk-heading-l">{{ page_title }}</h1>
     </div>
   </div>
   {% if multiple_choice_questions %}
-    {% for question, responses in multiple_choice_questions.items() %}
+    {% for mutliple_choice_question, multiple_choice_responses in multiple_choice_questions.items() %}
```

Comment:
Typo in "mutliple_choices_question"

Reply:
Doh! Well at least I made the typo consistently everywhere for `mutliple_choice_question`. Thanks for flagging.

Reply:
Ha ha, yes, at least you were consistent! (and it worked!)